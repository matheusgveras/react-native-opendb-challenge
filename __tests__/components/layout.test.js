import 'react-native';
import React from 'react';
import Layout  from '../../src/components/layout';
import {Text} from 'react-native'
import renderer from 'react-test-renderer';
describe('testing layout', () => {
    
    it('renders layout loading: false', () => {
        const view = <Layout loading={false}><Text>teste</Text></Layout>
        const tree = renderer.create(view).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders layout loading: true', () => {
        const view = <Layout loading={true} />
        const tree = renderer.create(view).toJSON();
        expect(tree).toMatchSnapshot();
    });
});