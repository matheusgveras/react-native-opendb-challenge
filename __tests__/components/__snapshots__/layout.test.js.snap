// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing layout renders layout loading: false 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#cccccc",
      "flex": 1,
    }
  }
>
  <Text>
    teste
  </Text>
</View>
`;

exports[`testing layout renders layout loading: true 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#cccccc",
      "flex": 1,
    }
  }
>
  <Text>
    Carregando...Carregando...Carregando...Carregando... Carregando...Carregando...Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando... Carregando...
  </Text>
</View>
`;
